media-video/ffmpeg/ffmpeg6-6.1.2.recipe

```diff
@@ -6,15 +6,15 @@ HOMEPAGE="https://ffmpeg.org/"
 COPYRIGHT="2000-2003 Fabrice Bellard
 	2003-2024 the FFmpeg developers"
 LICENSE="GNU LGPL v3"
-REVISION="3"
+REVISION="4"
 SOURCE_URI="https://ffmpeg.org/releases/ffmpeg-$portVersion.tar.xz"
 CHECKSUM_SHA256="3b624649725ecdc565c903ca6643d41f33bd49239922e45c9b1442c63dca4e38"
 PATCHES="ffmpeg-$portVersion.patchset"
 SOURCE_DIR="ffmpeg-$portVersion"
 ADDITIONAL_FILES="gcc_runtime.c"
 
 ARCHITECTURES="all"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
```

Comment:
I know there is no target for this, but it's pointless at this stage to remove it (in ffmpeg7 also), so please revert.